test/e2e/scheduling/nvidia-gpus.go

```diff
@@ -71,6 +71,16 @@ func makeCudaAdditionDevicePluginTestPod() *v1.Pod {
 						},
 					},
 				},
+				// CUDA 10.1
+				{
+					Name:  "vector-addition-cuda10-1",
+					Image: imageutils.GetE2EImage(imageutils.CudaVectorAdd3),
+					Resources: v1.ResourceRequirements{
+						Limits: v1.ResourceList{
+							gpuResourceName: *resource.NewQuantity(1, resource.DecimalSI),
+						},
+					},
+				},
```

Comment:
I think most of gpu tests are set up with 2 gpus on each machine, so running with three containers each requesting a gpu would make the gpu job not schedulable. I guess we perhaps can replace cuda8 container with cuda10.1 now.

Reply:
Would we want to increase the number of gpus to 3 for the tests in order to make sure one older version still works, or is testing 10 and 10.1 sufficient?